addons/controls/README.md

```diff
@@ -84,23 +84,39 @@ export const Reflow = () => {
   const label = text('Label', 'reflow');
   return (
     <>
-      {Array.from({ length: count }, (_, i) => (
-        <Button label={`button ${i}`} />
+      {[...Array(count)].map((_, i) => (
```

Comment:
Why does it need to be spread at all?

```suggestion
      {Array(count).map((_, i) => (
```

Reply:
Spreading the array is an imperative step that cannot be skipped: creating Arrays such as `Array(3)` for example will not create an Array with 3 "places" and the javascript language forbids iterating such arrays because they are **empty**. It only has 3 placeholder pointers, but it itself is empty. 

By spreading it, the javascript language fills each placeholder pointer with `undefined` value, and then instead of nothing, now each item has "something" and it now becomes iterable, so the `map` method could be used.

Reply:
Awesome. Today I learned! 💯